src/functionaltests/java/com/ericsson/ei/utils/HttpPostRequests.java

```diff
@@ -0,0 +1,132 @@
+package com.ericsson.ei.utils;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.http.client.methods.*;
+import org.apache.http.entity.StringEntity;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class HttpPostRequests {
+    private static final Logger LOGGER = LoggerFactory.getLogger(HttpPostRequests.class);
+    private int port;
+    private String url = "http://localhost:";
```

Comment:
You haven't constructor by default, so values for fields url and endpoint unneeded